src/Symfony/Bundle/FrameworkBundle/Tests/Functional/CachePoolsTest.php

```diff
@@ -50,7 +60,7 @@ public function testRedisCustomCachePools()
         try {
             $this->doTestCachePools(array('root_config' => 'redis_custom_config.yml', 'environment' => 'custom_redis_cache'), RedisAdapter::class);
         } catch (\PHPUnit_Framework_Error_Warning $e) {
-            if (0 !== strpos($e->getMessage(), 'unable to connect to 127.0.0.1')) {
+            if (0 !== strpos($e->getMessage(), 'unable to connect to')) {
```

Comment:
Nothing :) i don't have a problem with the copy&paste at the moment. My problem is if tests fail to connect to redis they're not skipped. Not sure how to solve it yet, so I'll leave it for now.